components/overflow-group/overflow-group.js

```diff
@@ -494,16 +504,16 @@ class OverflowGroup extends RtlMixin(LocalizeCoreElement(LitElement)) {
 	}
 
 	_handleSlotChange() {
-		requestAnimationFrame(() => {
-			this._getItems();
+		requestAnimationFrame(async() => {
+			await this._getItems();
 
 			this._slotItems.forEach(item => {
 				const observer = new MutationObserver(this._handleItemMutation);
 				observer.observe(item, {
 					attributes: true, /* required for legacy-Edge, otherwise attributeFilter throws a syntax error */
-					attributeFilter: ['disabled', 'text'],
+					attributeFilter: ['disabled', 'text', 'selected'],
```

Comment:
These changes were made due to filter - we want to trigger the mutation observer when the `selected` attribute changes in the `d2l-filter-dimension-set-value`s